lib/api/controllers/serverController.js

```diff
@@ -315,6 +318,34 @@ class ServerController extends NativeController {
       throw kerror.get('api', 'assert', 'unexpected_argument', scope, ['kuzzle', 'app', 'all']);
     }
 
+    if (exclude.length) {
+      definition.tags = definition.tags.filter(tag => ! exclude.includes(tag.name));
+      Object.entries(definition.paths).forEach(([path, pathDefinition]) => {
+        Object.entries(pathDefinition).forEach(([method, methodDefinition]) => {
+          if (methodDefinition.tags.some(tag => exclude.includes(tag))) {
+            delete pathDefinition[method];
```

Comment:
Avoid the `delete` keyword because it causes change in the object hidden class tree 

(you can read about [hidden classes(https://www.freecodecamp.org/news/javascript-essentials-why-you-should-know-how-the-engine-works-c2cc0d321553/) and [inline cache](https://mrale.ph/blog/2015/01/11/whats-up-with-monomorphism.html))